PlaylistManager/manifest.json

```diff
@@ -3,18 +3,18 @@
     "id": "PlaylistManager",
     "name": "PlaylistManager",
     "author": "PixelBoom",
-    "version": "1.7.2",
+    "version": "1.7.3",
     "description": [
         "#![PlaylistManager.Description.md]",
         "Load, Create, Edit, Organize Playlists and Download their Songs in Beat Saber."
     ],
-    "gameVersion": "1.37.3",
+    "gameVersion": "1.37.4",
     "icon": "PlaylistManager.Icons.Logo.png",
     "dependsOn": {
-        "BSIPA": "^4.3.0",
-        "SongCore": "^3.14.8",
-        "BeatSaberMarkupLanguage": "^1.7.5",
-        "SiraUtil": "^3.0.0",
+        "BSIPA": "^4.3.5",
+        "SongCore": "^3.14.13",
+        "BeatSaberMarkupLanguage": "^1.12.0",
+        "SiraUtil": "^3.1.12",
```

Comment:
Would prefer that you only bump dependencies that actually got breaking changes.

Reply:
I reversed, but kept BSML 1.20.0+ since that's a breaking change. As for SongCore, I got no idea.

Reply:
Nothing broke with 1.38.0, so should keep `gameVersion` as-is.

Reply:
Oh alright, well I'll revert to 1.37.4.